pdns/webserver.cc

```diff
@@ -281,13 +281,14 @@ try {
   HttpRequest req;
   YaHTTP::AsyncRequestLoader yarl;
   yarl.initialize(&req);
+  req.max_request_size=d_maxbodysize;
   int timeout = 5;
   client->setNonBlocking();
 
   try {
     while(!req.complete) {
       int bytes;
-      char buf[1024];
+      char buf[16000];
```

Comment:
I wonder if we should make this configurable (and in that case perhaps move it to the heap?). This should be fine on most OS, though.

Reply:
At 16000 my impression is that actual processing of the request will be a way bigger part of execution time anyway, so I'm not sure we need yet another config option.